src/Operator/Constructors.jl

```diff
@@ -532,10 +662,177 @@ function diffusionoperator(basis::AbstractBasis, mesh::Mesh)
     return diffusion
 end
 
+"""
+```julia
+advectionoperator(basis, mesh)
+```
+
+Convenience constructor for advection operator
+
+# Weak form:
+- ``\\int v u`
```

Comment:
is this the right weak form?

Reply:
For advection: 
We have $$ \int v * u_x $$

Reply:
So the comment above is incorrect, right? Do you feel comfortable with Git to make the requested changes and squash into your commits?